hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -0,0 +1,590 @@
+          resval: "0x0"
+        },
+        { bits: "26",
+          name: "CSAAT",
+          desc: '''Chip select active after transaction.  If CSAAT = 0, the
+                   chip select line is raised immediately at the end of the
+                   transaction.   If CSAAT = 1, the chip select line is left
+                   low at the end of the current sequence.  This potentially
+                   allows for the creation of longer or otherwise more
+                   complex sequences.'''
+          resval: "0x0"
+        },
+        { bits: "25:23",
+          name: "CSNLEAD",
+          desc: '''CS_N Leading Time.  Indicates the number of half sck cycles,
+                   CSNLEAD+1, to leave between the falling edge of cs_n and
+                   the first edge of sck.  Setting this register to zero
+                   corresponds to the minimum delay of one-half sck cycle'''
+          resval: 0
+        },
+        { bits: "22:20",
+          name: "CSNTRAIL"
+          desc: '''CS_N Trailing Time.  Indicates the number of half sck cycles,
+                   CSNTRAIL+1, to leave between last edge of sck and the rising
+                   edge of cs_n. Setting this register to zero corresponds
+                   to the minimum delay of one-half sck cycle.'''
+          resval: 0
+        },
+        { bits: "19:17",
+          name: "CSNIDLE"
+          desc: '''Minimum idle time between commands. Indicates the minimum
+                   number of sck half-cycles to hold cs_n high between commands.
+                   Setting this register to zero creates a minimally-wide CS_N-high
+                   pulse of one-half sck cycle.'''
+          resval: 0
+        },
+        { bits: "15:0",
+          name: "CLKDIV"
+          desc: '''Core clock divider.  Slows down subsequent SPI transactions by a
+                   factor of (CLKDIV+1) relative to the core clock frequency.  The
+                   period of sck, T(sck), then becomes 2*(CLK_DIV+1)*T(core)'''
+          resval: 0
+        },
+      ]
+    },
+    { name: "COMMAND",
+      desc: "Command Register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "3:0",
+          name: "TX1_CNT",
+          desc: '''The number of 1-byte bursts to transmit in this command,
+                   at the single-line data width (8 sck clocks per cycle).
+                   All other data lines are high-Z during this command phase.
+                   This field is ignored when SPEED = 0.
+                   ''',
+          resval: "0x0"
+        },
+        { bits: "12:4",
+          name: "TXN_CNT",
+          desc: '''The number of 1-byte bursts to transmit in this command,
+                   at the full-data width (Determined by the SPEED field).
+                   ''',
+          resval: "0x0"
+        },
+        { bits: "16:13",
+          name: "DUMMY_CNT",
+          desc: '''Controls the number of sck dummy cycles (NOT bytes)
```

Comment:
I'm not sure that the "NOT bytes" bit is necessary here. The documentation is pretty clear about how this all works so I don't think we need anything unless this looks like another command interface that uses bytes. How about renaming the field to e.g. DUMMY_CYCLES?

Reply:
I like the rename.  Changed it here and in the md.